src/test/java/com/endava/internship/mocking/service/BasicValidationServiceTest.java

```diff
@@ -0,0 +1,74 @@
+package com.endava.internship.mocking.service;
+
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+class BasicValidationServiceTest {
```

Comment:
Here you have only negative flows, please add and positive to the existing tests

Reply:
Added new methods to check for valid values. True, I left the "Then" block without code. I'm not sure it makes sense to check for void methods. My idea was to create a Spy class so that I could use verify. But why? If the method does not return anything, it can only throw an exception, and then it is clear that the test has failed.